### PR TITLE
Send failing emails to an SQS queue

### PR DESCRIPTION
**Pull request by keirog**
If the email service returns < 200 or > 299:
- if 410, send the event data to the unsubscribe queue
- otherwise send to the retry queue

Sorry for the naff PR 😷  😰

### CODE REVIEW

File: functions/send-email/lib/build-request-options.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	headers: {
+		'Authorization': process.env.EMAIL_SERVICE_ACCESS_KEY,
+		'Content-Type': 'application/json'
+	},
+	body: {
+		transmissionHeader: {
+			metadata: {}
+		},
+		from: {
+			address: 'myft@news-alerts.ft.com',
```
Comment (laurieboyes) [inline functions/send-email/lib/build-request-options.js]: Will this be a real email address?

File: functions/send-email/test/lib/send-to-queue.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		sendToQueue(createError('I’m a teapot', { status: 418, event }));
+		sinon.assert.calledOnce(sendToQueueMock.queueRetry);
+	});
+
+	it('doesn’t call queueUnsubscribe if the error status is a non-410 error', () => {
+		sendToQueue(createError('I’m a teapot', { status: 418, event }));
+		sinon.assert.notCalled(sendToQueueMock.queueUnsubscribe);
+	});
+
+	it('blah', () => {
```
Comment (laurieboyes) [inline functions/send-email/test/lib/send-to-queue.spec.js]: image

**Comment by laurieboyes** [comment]
It's possible we'll change this to use Kineses streams instead, but merging in for now

File: functions/send-email/lib/build-request-options.js (hunk 1 above)
Comment (keirog) [inline functions/send-email/lib/build-request-options.js]: Hopefully this is the real one—I took it from [the old app](https://github.com/Financial-Times/next-myft-email-generator/blob/f8a30001fdf59ef56661e8b72f4efe8f171ad132/jobs/create-email.js#L40)

Comment (phamann) [inline functions/send-email/lib/build-request-options.js]: Does this mean that @keirog is back?

**Pull request merged**